src/test/java/com/codeus/winter/config/DefaultBeanFactoryTest.java

```diff
@@ -0,0 +1,203 @@
+package com.codeus.winter.config;
+
+import com.codeus.winter.exception.BeanNotFoundException;
+import com.codeus.winter.exception.NotUniqueBeanDefinitionException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class DefaultBeanFactoryTest {
+    /**
+     * Testing class.
+     */
+    private DefaultBeanFactory beanFactory;
+
+    @BeforeEach
+    void setUpBeforeClass() {
+        beanFactory = new DefaultBeanFactory();
+    }
+
+    /**
+     * Test {@link DefaultBeanFactory#getBean(String)} with {@code name}
+     * where bean with 'name' is missing.
+     * Method throws BeanNotFoundException.
```

Comment:
Documentation of test methods is redundant

Reply:
I agree. 
But I can't find the way to exclude the checking for all test classes.
Documentation tells us about this case:
https://checkstyle.org/filters/suppressionxpathfilter.html 
Quotes: "the filter does not support suppressions inside javadoc reported by Javadoc checks:

- AtclauseOrder
- JavadocBlockTagLocation
- JavadocMethod
- JavadocMissingLeadingAsterisk
- JavadocMissingWhitespaceAfterAsterisk
- JavadocParagraph
- JavadocStyle
- JavadocTagContinuationIndentation
- JavadocType
- MissingDeprecated
- NonEmptyAtclauseDescription
- RequireEmptyLineBeforeBlockTagGroup
- SingleLineJavadoc
- SummaryJavadoc
- WriteTag

Note, that support for these Checks will be available after resolving issue [#5770](https://github.com/checkstyle/checkstyle/issues/5770)"

If somebody knows how to resolve this case, I will be genuinely grateful.